### DISCUSSION TITLE
Add links to runner output that open the file directly in my IDE

### DISCUSSION THREAD

**Discussion by ThomasLandauer** [Codeception/Codeception#6208] [Ideas]
I just added a nice feature to PHPStan: When you set an `editorUrl`, the output will get an additional line with a link to open the "offending" file directly in your IDE. See https://phpstan.org/user-guide/output-format#opening-file-in-an-editor for how this looks like. The idea is taken from Symfony, see https://symfony.com/doc/current/reference/configuration/framework.html#ide

So I'm suggesting the same for Codeception: The option to add a link to the output to open the Cest/Test file.

**Comment by Naktibalda** [comment]
Do you want to add links to HTML reports?

**Reply by ThomasLandauer** [reply]
No, but a link to open the failing Cest file. So the end result would look something like this:
```
phpstorm://open?file=/home/dev/tests/MyCest.php&line=30
```
When you register the protocol in your system, this will become a clickable link in the terminal.

I know that, if a test fails, the actual problem is sometimes in the test, and sometimes in the code. Anyway, I am opening the test file almost every time, to see what's going on where it fails. So having a direct link there would be a handy time-saver.

**Reply by Naktibalda** [reply]
It looks useful, can you implement it?

**Reply by ThomasLandauer** [reply]
I started, but I didn't get very far yet ;-) See https://github.com/Codeception/Codeception/issues/6259

**Reply by ThomasLandauer** [reply]
Pull request: https://github.com/Codeception/Codeception/pull/6261
